keycloak/helm/keycloak/templates/deployment.yaml

```diff
@@ -0,0 +1,61 @@
+apiVersion: apps/v1
```

Comment:
all of these scaffold helm files can probably be removed in favor of the subchart.